service/src/geopm/Exception.hpp

```diff
@@ -119,6 +119,11 @@ namespace geopm
             /// The error code associated with the exception.
             int m_err;
     };
+
+    /// @param error_value The error code associated with the exception.
```

Comment:
```suggestion
    /// @brief Function that converts an error code into an error message
    /// @param error_value The error code associated with the exception.
```

Reply:
There is an equivalent C interface here:

https://github.com/geopm/geopm/blob/dev/service/src/geopm_error.h#L60

perhaps it would be best to match the name:

geopm::get_error_string -> geopm::error_message()

Reply:
The interface Chris is referencing is implemented here:
https://github.com/geopm/geopm/blob/dev/service/src/Exception.cpp#L71

@cmcantalupo Why have two different implementations though?  My thought was to change the test to use the C interface and remove this one.

Reply:
It is useful to be able to directly use the std::string type and not have a fixed length C string.  Additionally the C interface provides the message from the last handled exception (useful for our external C interfaces), but this is not helpful in a context where there may not be a handled exception.